package.json

```diff
@@ -26,11 +26,11 @@
     "promise-map-series": "^0.2.1"
   },
   "devDependencies": {
-    "broccoli-fixture": "^1.0.0",
+    "broccoli-fixture": "^0.1.0",
```

Comment:
This seems strange, why would we roll back to 0.1.0 here?

Reply:
no idea, reverted it though.